benches/bench.rs

```diff
@@ -57,6 +57,7 @@ use std::collections::HashMap;
 use std::convert::{TryFrom, TryInto};
 
 const INPUT_SIZES: [u16; 8] = [1, 2, 8, 16, 32, 64, 128, 256];
+const INPUT_SIZES_ARITHMETIC: [u16; 8] = [1, 2, 3, 4, 5, 6, 7, 8];
```

Comment:
only concern here is that these sizes are too small to get a good idea of the constants - but we can leave it for now and revisit if the benchmarks end up strange

Reply:
agreed, I think we might want to bump these up a bit

Reply:
keeping this as is for now... when I go above 8 I start seeing arithmetic overflows. Perhaps we should use smaller numbers in the generator? Part of me feels like if we can get a good fit on the output, these small sizes are actually more representative of how these functions will be used...